### PR TITLE
[CI] Fix a description bug for AddSelfLoop

### PR DESCRIPTION
**Pull request by caojy1998**
## Description

Correct an erroneous case for AddSelfLoop function.
Modify in file: dgl/python/dgl/transforms/module.py

## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] I've leverage the [tools](https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit) to beautify the python and c++ code.
- [ ] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

File: python/dgl/transforms/module.py [hunk 1]
```diff
@@ -544,7 +544,7 @@ class AddSelfLoop(BaseTransform):
      >>> g.edata['he'] = torch.arange(3).float().reshape(-1, 1)
      >>> new_g = transform(g)
      >>> print(new_g.edges())
-     (tensor([1, 0, 1, 2]), tensor([2, 0, 1, 2]))
+     (tensor([0, 0, 2, 0, 1, 2]), tensor([2, 1, 0, 0, 1, 2]))
```
Comment (frozenbugs) [inline python/dgl/transforms/module.py:547]: Can you also fix the issues on the other doc string for this method?

e.g. L560 to L567 is misaligned, such that the render [under case 2](https://docs.dgl.ai/generated/dgl.transforms.AddSelfLoop.html) is broken.

Comment (caojy1998) [reply python/dgl/transforms/module.py:547]: OK, I will try to fix them.

**Comment by caojy1998** [comment]
@dgl-bot

File: python/dgl/transforms/module.py [hunk 2]
```diff
@@ -557,19 +557,19 @@ class AddSelfLoop(BaseTransform):
...
-    >>> g.edata['feat1'] = {('user', 'follows', 'user'): torch.randn(2, 15),
-      ...                   ('user', 'plays', 'game'): torch.randn(2, 15)}
+     ...     ('user', 'follows', 'user'): (torch.tensor([1, 2]),
+     ...                                   torch.tensor([0, 1])),
+     ...     ('user', 'plays', 'game'): (torch.tensor([0, 1]),
+     ...                                 torch.tensor([0, 1]))})
+     >>> g.edata['feat'] = {('user', 'follows', 'user'): torch.randn(2, 5),
+     ...                    ('user', 'plays', 'game'): torch.randn(2, 5)}
+     >>> g.edata['feat1'] = {('user', 'follows', 'user'): torch.randn(2, 15),
+     ...                    ('user', 'plays', 'game'): torch.randn(2, 15)}
```
Comment (frozenbugs) [inline python/dgl/transforms/module.py:567]:
```suggestion
     >>> g.edata['feat1'] = {('user', 'follows', 'user'): torch.randn(2, 15),
     ...                     ('user', 'plays', 'game'): torch.randn(2, 15)}
```

Comment (caojy1998) [reply python/dgl/transforms/module.py]: Still fail after apply the suggested change.

Comment (caojy1998) [reply python/dgl/transforms/module.py]: solve this in 5843

**Review by frozenbugs — APPROVED** [review]

**Pull request merged**